src/mimeparser.rs

```diff
@@ -232,7 +235,7 @@ impl MimeMessage {
         hop_info += &decryption_info.dkim_results.to_string();
 
         // `signatures` is non-empty exactly if the message was encrypted and correctly signed.
-        let (mail, signatures, warn_empty_signature) =
+        let (mail, signatures, encrypted) = // keep formatting
```

Comment:
rustfmt is really evil, it wants to reformat the entire `let` block w/o this comment